script/fs-bench/work/tools/scrape/main.go

```diff
@@ -0,0 +1,186 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
+	"syscall"
+	"time"
+
+	proc "github.com/c9s/goprocinfo/linux"
```

Comment:
Why don't you use https://github.com/prometheus/procfs ? This seems more actively maintained.

Reply:
c9s/goprocinfo has a simpler API than prometheus/procfs'

Reply:
But we need to prioritize maintainability over the API simplicity. Can you use that package?